libexec/rtld-elf/aarch64/reloc.c

```diff
@@ -103,12 +103,6 @@ init_cap_from_fragment(void *where, void * __capability data_cap,
 		cap = cheri_setbounds(cap, len);
 	}
 
-	if (perms == MORELLO_FRAG_EXECUTABLE && addend != 0) {
```

Comment:
And there was much rejoicing...

Reply:
The dynamic symbol ones still exist though... still need to get to the bottom of why those _aren't_ dynamic for us, only Morello, as they _should_ be dynamic for us too...